Source/options.js

```diff
@@ -512,6 +512,16 @@ $(function() {
     save('ngramMax', 'Maximum word sequence length saved');
   }
 
+  function validate_yandexTranslatorApiKey(){
+    var yandexKey = e("yandexTranslatorApiKey").value
+    if (yandexKey.match(/Unblock|Block/)){
+      alert("Please enter valid Yandex key.\nRemove Block key / Unblock key from the end of the key")
```

Comment:
Yes, @sjoanes is right. The error message in case of 'Unblock' should probably be something like this: "This key has been blocked. Please go to Yandex and create a new key."